### PR TITLE
Created Initial Test Plan documents

### PR DESCRIPTION
**Pull request by kathweaver**
Going for it.  Adding the first two Test Plan documents

### CODE REVIEW

**Comment by kathweaver** [comment]
This is the beginning of the resolution for Issue #162

File: TestPlan/InstallAsNew [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#Test Plan for Ensuring that Installing as an Update works properly
```
Comment (willduff) [inline TestPlan/InstallAsNew]: These files don't have a file extension, so GitHub doesn't know they are Markdown files. Can you change these to have a *.md file extension?

**Comment by kathweaver** [comment]
Renamed files and fixed some typographical errors.

**Comment by kathweaver** [comment]
Added to the test procedures document.  Steps so that testers can test from SetupOpenWriter.exe and not have to pull and build code.

**Pull request merged**